GoogleSignIn/Sources/GIDSignIn.m

```diff
@@ -283,11 +283,9 @@ - (void)addScopes:(NSArray<NSString *> *)scopes
     }
     return;
   }
-
-  // Use the union of granted and requested scopes.
-  [grantedScopes unionSet:requestedScopes];
-  options.scopes = [grantedScopes allObjects];
-
+  
+  options.scopes = [requestedScopes allObjects];
```

Comment:
Maybe add comment here that says that we pass `include_granted_scopes=true` here so the previously granted scopes will still be there.

Reply:
Done.